module/common/Logging.h

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file Logging.h
+ * @brief 動作確認に用いるprintf()関数を所持するクラス
+ * @author sap2368
+ */
+#ifndef LOGGING_H
+#define LOGGING_H
+
+#include <stdio.h>
+
+class Logging {
+ public:
+  Logging();
+
+  /**
+  ** 入力された文字列をLogとしてターミナルに表示
+  ** @param logMessage
+  **/
```

Comment:
すみません，説明不足でした．
- 最初の行の*以外は一つでいい
- 「@param 変数名 変数の説明」の形で書く
- ヘッダファイルに書いてある他の関数宣言においても同様
- [参考記事](https://qiita.com/inabe49/items/23e615649e8539d857a8)

他のクラスのヘッダファイルを参考にするといいと思います．なんならコピペして書き換えるの方が確実かもしれないです．

```suggestion
  /**
  * 入力された文字列をLogとしてターミナルに表示
  * @param logMessage 表示するLogメッセージ
  */
```